### PR TITLE
set jpeg thumbnail quality to 35

### LINKED ISSUE #16692: set jpeg thumbnail quality
In #16639 we were supposed to set the jpeg quality to 35 to get the file size vs. quality at half scale trick to work, but that detail got lost along the way in the slack discussions.  We need to do that.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #16692

### CODE REVIEW

File: src/olympia/amo/utils.py [hunk 1]
```diff
@@ -666,12 +666,14 @@ def pngcrush_image(src, **kw):
     return False
 
 
-def resize_image(source, destination, size=None, format='png'):
+def resize_image(source, destination, size=None, *, format='png', quality=80):
```
Comment (championshuttler) [inline src/olympia/amo/utils.py:669]: Sorry if i am wrong but why you setting it to 80?

Comment (championshuttler) [inline src/olympia/amo/utils.py:669]: Sorry if i am wrong but why you setting it to 80? is it going to be ignored for jpg (quality=80)?

Comment (eviljeff) [reply src/olympia/amo/utils.py:669]: 80 is the default value of `quality` if the kwarg isn't provided.  As to "why 80?" - 80 is generally the best compromise between filesize and not losing any noticeable quality. Mainly though, it's been the default we've used for years for jpegs so I didn't want to regress the use of `resize_image` elsewhere.

Comment (championshuttler) [reply src/olympia/amo/utils.py:669]: thank you so much @eviljeff got it, sorry for noise

**Review by willdurand — APPROVED** [review]

**Pull request merged**